dashboard/test/controllers/pd/workshop_dashboard_controller_test.rb

```diff
@@ -1,21 +1,21 @@
 require 'test_helper'
 
 class Pd::WorkshopDashboardControllerTest < ::ActionController::TestCase
-  test 'admins can access the dashboard' do
+  test 'admins can access the dashboard and get correct permission value' do
```

Comment:
Should these be two separate tests, one using the framework you are building? Alternately, change `generate_user_tests_for` to accept an optional block that gets `yield`ed to, putting the additional `assert_equal` in that block? And I'm really not sure I'm suggesting that, though I do feel it is a common paradigm.